src/main/resources/assets/minestuck/blockstates/lunchtop.json

```diff
@@ -8,9 +8,9 @@
         "facing=east,state=on":      { "model": "minestuck:block/lunchtop_on", "y": 90 },
         "facing=south,state=on":     { "model": "minestuck:block/lunchtop_on", "y": 180 },
         "facing=west,state=on":      { "model": "minestuck:block/lunchtop_on", "y": 270 },
-        "facing=north,state=broken": { "model": "minestuck:block/lunchtop_bsod" },
-        "facing=east,state=broken":  { "model": "minestuck:block/lunchtop_bsod", "y": 90 },
-        "facing=south,state=broken": { "model": "minestuck:block/lunchtop_bsod", "y": 180 },
-        "facing=west,state=broken":  { "model": "minestuck:block/lunchtop_bsod", "y": 270 }
```

Comment:
Is this change from "bsod" to "off" intentional?

Reply:
theres no screen, so no bsod texture anymore. Ive put the off state instead

Reply:
For this, I think it's more appropriate that the "on" model is used for the broken state, since a "broken" computer is a variant of the "on" state when it comes to functionality.